source/mapping-rules/mapping-rule-options/time-series.txt

```diff
@@ -0,0 +1,122 @@
+.. _rm-timeseries:
+
+===========
+Time Series
+===========
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+You can configure mapping rules to migrate a table that contains a date 
+time column as a :ref:`time series collections <manual-timeseries-collection>`. 
```

Comment:
[question]

I typically see "datetime" written as a single word. If it appears as two words in the UI, let's leave as two.